### ISSUE TITLE
ERROR in ./node_modules/xpath-helper/dist/mjs/xpath-helper.js [SOLVED]

### ISSUE THREAD

**Issue by nopmop** [jrebecchi/xpath-helper#4]
some interesting error appears when importing in a commonjs based project:

```ERROR in ./node_modules/xpath-helper/dist/mjs/xpath-helper.js 9:6
Module parse failed: Unexpected token (9:6)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  */
| export class XPathHelper {
>     sb;
|     /**
|      * Creates an instance of XPathHelper.
 @ ./src/modules/module/index.ts 9:0-42 50:17-19 50:41-47
 @ ./src lazy ^\.\/.*$ namespace object
 @ ./src/content_scripts/content.tsx
```
I tried all possible combinations of options in tsconfig and webpack but nothing worked. It seems as if `mjs` has precedence over `cjs` or that `mjs` is set as default somewhere so at the end I resorted to deleting `./node_modules/xpath-helper/dist/mjs/`. That caused `./node_modules/xpath-helper/dist/cjs/` to be loaded.

**Comment by jrebecchi** [comment]
Thanks, @nopmop for raising the issue :)
Would you have a project where I can reproduce the issue? 
I wonder if adding `"default": "./dist/cjs/xpath-helper.js",` property inside `exports` object of `package.json` solve the issue.

**Comment by nopmop** [comment]
I prepared a project skeleton for you (`jrebecchi.zip`). 
To test, unzip, `cd jrebecchi`, then do `npm i; npm run watch` - you will see the error.
Then do: `rm -rf ./node_modules/xpath-helper/dist/mjs/` and run `npm run watch` again, you'll see the error is gone.
It's good to see you're providing support actively for your library :)
Cheers!

[jrebecchi.zip](https://github.com/jrebecchi/xpath-helper/files/9747984/jrebecchi.zip)

**Comment by jrebecchi** [comment]
@nopmop fix published in version [v0.1.3](https://github.com/jrebecchi/xpath-helper/releases/tag/v0.1.3) :)
